projects/1.18/assets/construction-wand/constructionwand/lang/zh_cn.json

```
@@ -4,16 +4,21 @@
   "item.constructionwand.diamond_wand": "钻石手杖",
   "item.constructionwand.infinity_wand": "无尽手杖",
   "item.constructionwand.core_angel": "天使手杖核心",
+
   "item.constructionwand.core_destruction": "破坏手杖核心",
   "constructionwand.tooltip.blocks": "最多放置 %d 个方块",
   "constructionwand.tooltip.shift": "按住 [SHIFT]",
   "constructionwand.tooltip.cores": "手杖核心:",
+  "constructionwand.tooltip.core_tip": "在一个合成格中将核心与你的魔杖结合在一起。",

```

唯一的修改1

---

```suggestion
  "constructionwand.tooltip.core_tip": "在合成方格中将核心与你的手杖合成到一起",
```
没有句号
原文：Combine the core with your wand in a **crafting grid**
https://minecraft.fandom.com/zh/wiki/%E5%90%88%E6%88%90#.E5.90.88.E6.88.90.E6.96.B9.E6.A0.BC